ontimize/components/input/hour-input/o-hour-input.component.ts

```diff
@@ -244,6 +266,16 @@ export class OHourInputComponent extends OFormDataComponent implements OnInit, A
     return result;
   }
 
+  public getValue(): any {
+    let value = super.getValue();
+    if (this.valueType === 'timestamp') {
+      if ( !isNaN(moment(value, this.formatString).valueOf()) ) {
+        value = moment(value, this.formatString).valueOf();
```

Comment:
Instruction repeated `moment(value, this.formatString).valueOf()`, store it in a variable and use that variable afterwards